src/common/playideas/ideas.json

```diff
@@ -123,7 +123,13 @@
     {
       "id": "idea-21",
       "title": "Animations Portfolio",
-      "description": "Build a Portfolio made with react and sanity by framer-motion. Projects, Portfolio Section, Skills & Experience, Companies you work in with carousel, and Contacts.",
+      "description": "Build a Portfolio made with react by framer-motion. Projects, Portfolio Section, Skills & Experience, Companies you work in with carousel, and Contacts.",
```

Comment:
What's the difference between these two projects? Looks almost similar to me.

Reply:
Can you make it a little more clear. The difference in the descriptions are not very clear. It looks almost similar to me(and someone else told the same)

Reply:
> Can you make it a little more clear. The difference in the descriptions are not very clear. It looks almost similar to me(and someone else told the same)

Ok